src/Renderer/ThreeExtended/GLTFLoader.js

```diff
@@ -80,8 +80,6 @@ THREE.GLTFLoader = ( function () {
 
 			}
 
-			console.time( 'GLTFLoader' );
```

Comment:
How about surrounding these with `if(__DEBUG__)` instead of removing them?

Reply:
Because it's very noisy when you have a big tileset.

Is this timing info important for you?

Reply:
It has been useful a few times, but I'm starting to work with glTF2, you can go ahead and delete it.